dashboard/lti_new.py

```diff
@@ -118,10 +118,15 @@ def extracting_launch_variables_for_tool_use(request, message_launch):
     roles = launch_data['https://purl.imsglobal.org/spec/lti/claim/roles']
     username = custom_params['user_username']
     course_id = custom_params['canvas_course_id']
+    canvas_course_long_id = canvas_id_to_incremented_id(course_id)
+    canvas_user_id = custom_params['canvas_user_id']
+    canvas_user_long_id = canvas_id_to_incremented_id(canvas_user_id)
     email = launch_data['email']
     first_name = launch_data['given_name']
     last_name = launch_data['family_name']
+    user_name = launch_data['name']
```

Comment:
Nit-pick: variable name could be clearer here, it's super close to `username`, which already exists.